recipes/qt/5.x.x/patches/android-backtrace.diff

```diff
@@ -0,0 +1,19 @@
+diff --git a/src/corelib/global/qlogging.cpp b/src/corelib/global/qlogging.cpp
```

Comment:
Do you have have a link to an issue on bugreports.qt.io or to a patch on codereview.qt-project.org?
This level of android api should supported out of the box https://doc.qt.io/qt-5/android.html

Reply:
unfortunately, no such issue has been raised yet. I've also checked backports in https://invent.kde.org/qt/qt/qtbase/-/commits/kde/5.15 but there's also no such fix.

according to https://doc.qt.io/qt-5/android-getting-started.html Qt 5.15 has been tested only with NDK 20b and 21, pretty sure those toolchain versions don't have `backtrace` functions or `execinfo.h`.

Reply:
ok, thanks for the clarifications